app/controllers/react_controller.rb

```diff
@@ -1,3 +1,15 @@
 class ReactController < ApplicationController
   layout 'layouts/react_application'
+
+  def index
+    render 'react/react_page'
+  end
+
+  def new
```

Comment:
imho new/edit etc all should be handled by react router, e.g. /audits and /audits/* should be handled fully by the router (and not on rails side). I understand this conflicts with fetching audits via /audits in json format, but i think we should not use this pattern.